src/ts/show.ts

```diff
@@ -386,10 +402,10 @@ document.addEventListener('DOMContentLoaded', () => {
         // get the width of the parent. The -30 is to make it smaller than parent even with the margins
         const width = $('#parent_' + randId).width() - 30;
         // add html content and adjust the width of the children
-        const div = document.getElementById(randId);
         div.innerHTML = data.msg;
         div.style.width = String(width);
         div.toggleAttribute('hidden');
+        div.setAttribute('data-bodyLoaded', '1');
```

Comment:
It's ends up with the same result, but to me it seems better to do `div.dataset.bodyLoaded = '1'`.